src/qibo/quantum_info/metrics.py

```diff
@@ -574,7 +574,6 @@ def diamond_norm(channel, target=None, backend=None, **kwargs):
 
     .. note::
         This function requires the optional CVXPY package to be installed.
-
     """
     import cvxpy  # pylint: disable=C0415
```

Comment:
```suggestion
    import cvxpy
```
is the exception still needed since you added `cvxpy` as pylint ignored module in pyproject.toml? I am not sure.

Reply:
Most likely not, let me try to remove it.